packages/react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -146,12 +149,14 @@ export class DetailsColumnBase extends React.Component<IDetailsColumnProps> {
                 <span
                   id={`${parentId}-${column.key}`}
                   className={classNames.cellTitle}
-                  data-is-focusable={column.columnActionsMode !== ColumnActionsMode.disabled}
+                  data-is-focusable={
```

Comment:
Does a boolean not work properly here?

Reply:
The false value sets `tabindex="-1"` when false, making it focusable (but not tabbable), when ideally it has no tabindex and no focusability when disabled or using an inner button.